backend/test/prototype.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const MongoClient = require('mongodb').MongoClient
+
+const app = express()
+app.use(bodyParser.json())
+
+const url = 'mongodb://puru:19880818@ds155201.mlab.com:55201/test_new_mongodb'
+
+app.listen(3000, () => console.log('Listening to port 3000'))
+app.get('/user/:id', (req, res) => res.send(`Welcome to the homepage of user ${req.params.id}.`))
+
+const main = async () => {
+  const db = await MongoClient.connect(url)
+  // post news and comment
+  app.post('/data', async (req, res) => {
+    const data = req.body
+    data.timestamp = new Date()
+    data.upvote = 0
+    data.downvote = 0
+    data.comments = {}
+    console.log(data)
+    const r = await db.collection('Data').insertOne(data)
```

Comment:
There is a serious security flaw. Any object from the client of any shape will be inserted into the database!